[YUNIKORN-2349] Change the orientation of the queue

doupache opened the pull request:
### What is this PR for?
https://github.com/user-attachments/assets/0caa15fd-9089-4abf-8d7f-af7cb8cfede1


### What type of PR is it?
* [x] - Feature

### What is the Jira issue?
 [YUNIKORN-2349](https://issues.apache.org/jira/browse/YUNIKORN-2349)

doupache commented:
This PR builds upon the changes introduced in #189 and should be reviewed or merged after #189 has been completed.

craigcondit commented:
@doupache please rebase on master, #189 has been merged.

doupache commented:
This pr also contain the commit in #197.  Once we merge it , i will rebase this one

ryankert01 commented on src/app/utils/common.util.ts:
```diff
@@ -81,10 +81,47 @@ export class CommonUtil {
+      return '<strong>CPU:</strong> n/a';
+    }
+    let cpu = value.split('%')[1] + '%';
+    cpu = cpu.replace(',', '');
```
Maybe we can try catch, so whether or not `value` string in shape, it will return something and not crash.

example: 
```ts
try {
    const parsed = parseResourceValue(value);
    return `<strong>Memory:</strong> ${this.queueResourceColumnFormatter(parsed.memory)}`;
} catch (error) {
    console.error("Error parsing memory value:", error);
    return '<strong>Memory:</strong> Error';
}
```

brandboat commented:
gentle ping @doupache , could you help fix the conflicts ? Thanks

doupache commented:
Thank you @brandboat and @ryankert01 for the review. I've resolved the conflict in the master branch, addressed the incorrect resource format, and added test cases to ensure proper handling of the incorrect resource format.

doupache commented on src/app/utils/common.util.spec.ts:
```diff
@@ -58,4 +58,57 @@ describe('CommonUtil', () => {
+      expect(CommonUtil.absoluteUsedCPUColumnFormatter('CPU: 60%')).toBe('<strong>CPU:</strong> 60%');
+    });  
+    it('should handle input without percentage sign', () => {
+      expect(CommonUtil.absoluteUsedCPUColumnFormatter('60')).toBe('<strong>CPU:</strong> n/a (wrong cpu format)');
```
cc @ryankert01

doupache commented on src/app/utils/common.util.ts:
```diff
@@ -81,10 +81,59 @@ export class CommonUtil {
+      return '<strong>Memory:</strong> n/a';
+    }
+    //Memory: 4%, CPU: 2%
+    const memoryRegex = /Memory: ([0-9]|[1-9][0-9]|100)%/;
```
@ryankert01

doupache commented on src/app/utils/common.util.ts:
```diff
@@ -81,10 +81,59 @@ export class CommonUtil {
+      return '<strong>CPU:</strong> n/a';
+    }
+    //Memory: 4%, CPU: 2%
+    const cpuRegex = /CPU: ([0-9]|[1-9][0-9]|100)%/;
```
@ryankert01

ryankert01 reviewed (approved):
lgtm

craigcondit commented:
@doupache Can you rebase? Also, would it be possible to remove the animation on resize? It's quite slow.

craigcondit reviewed (approved):
+1 LGTM.
